### PR TITLE
Feat: Update Root README

### PR DESCRIPTION
**Pull request by chris-ingrisani**
This PR adds a few updates to the root README file.

### CODE REVIEW

**Comment by rutalreja-deloitte** [comment]
@googlebot I consent.

**Comment by chris-ingrisani** [comment]
Hi @bharathkkb, this PR is ready for review

File: README.md [hunk 1]
```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
 ## Instructions
```
Comment (bharathkkb) [inline README.md:73]: Why do we need these instructions?

File: README.md [hunk 2]
```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
...
+- **External IP** for accessing the application externally
+
+### [6. anthos-install](./6-anthos-install)
+
+The purpose of this step is to install the Anthos components required for the Bank of Anthos example application - Anthos Config Management and Anthos Service Mesh. **This step in the process is currently manual.**  Anthos Config Management can help you create a common configuration across all your infrastructure, including custom policies, and apply it both on-premises and across clouds. Anthos Service Mesh lets you easily manage the security and telemetry of complex environments.  Currently, this includes:
+
+- All required components to install **Anthos Config Management**
+- All required components to install **Anthos Service Mesh**
+
+### [7. build_app](./build_app)
```
Comment (bharathkkb) [inline README.md:181]: Where do we have the step by step instructions for pushing the source code to the repo to be built by this pipeline?

File: README.md [hunk 3]
```diff
@@ -65,15 +73,128 @@ href="https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tr
...
+   - Code coverage numbers are pulled to make a decision based on the results
+
+1. **Artifact Verification**
+   - Container structure tests to verify that the container built in the previous step conforms to organizational standards
+   - Container analysis to verify that the container does not contain Common Vulnerabilities and Exposures
+
+1. **Security Attestation**
+   - Creates an an attestation for the artifact
+
 ## Source Code Headers
```
Comment (bharathkkb) [inline README.md:198]: Lets remove this part

**Review by bharathkkb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rutalreja-deloitte) [reply README.md:181]: the step by step for the namespace repos exists in 6-anthos-install readme, the bank-of-anthos-source repo step by step will be in the build_app folder readme though currently atm it does not have them.

File: README.md (hunk 3 above)
Comment (rutalreja-deloitte) [reply README.md]: removed in 5ab064bf9e45ea280a7f72d103b9aec6e3fdfae2

File: README.md (hunk 1 above)
Comment (rutalreja-deloitte) [reply README.md]: removed in 5ab064bf9e45ea280a7f72d103b9aec6e3fdfae2

File: README.md (hunk 2 above)
Comment (bharathkkb) [reply README.md]: sg, lets open an issue to track the missing readme for that

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**